packages/victory-pie/src/index.d.ts

```diff
@@ -18,6 +18,8 @@ export type VictorySliceLabelPositionType = "startAngle" | "centroid" | "endAngl
 export type VictorySliceTTargetType = "data" | "labels" | "parent";
 
 export interface SliceProps extends VictoryCommonProps {
+  ariaLabel?: string;
```

Comment:
Since these will be on every primitive component, it might make sense to add them to `VictoryCommonProps`

Reply:
And here as well: https://github.com/FormidableLabs/victory/blob/main/packages/victory-core/src/victory-util/common-props.js#L147

Reply:
Great point, updated!